content/automate/ManageIQ/Cloud/VM/Provisioning/Placement.class/__methods__/best_fit_azure.rb

```diff
@@ -43,6 +43,7 @@ def default_cloud_network
 
               def default_cloud_subnet
                 cloud_subnet = prov.eligible_cloud_subnets.first
+                raise "No cloud subnets found for cloud network" unless cloud_subnet
 
                 if cloud_subnet
```

Comment:
@djberg96 This line becomes not necessary.

Reply:
@lfu updated, thanks. :)